src/routes/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { DEFAULT_AVATAR } from 'store/models/Community'
+import CommunityDetail from 'routes/CommunityDetail'
+import Icon from 'components/Icon'
+import Loading from 'components/Loading'
+import RoundImage from 'components/RoundImage'
+
+import './LandingPage.scss'
+
+export default class LandingPage extends Component {
+  static propTypes = {
+    community: PropTypes.object,
+    showAbout: PropTypes.func
+  }
+
+  static defaultProps = {
+    community: {},
+    showAbout: () => {}
+  }
+
+  componentDidMount () {
+  }
+
+  componentDidUpdate () {
+  }
+
+  render () {
+    const { community, location, match, showAbout } = this.props
+
+    if (!community) return <Loading />
+
+    const canView = community.memberCount > 0
```

Comment:
why is this needed? how could memberCount be 0 and someone be looking at this?

Reply:
Same comment as [above](https://github.com/Hylozoic/hylo-evo/pull/751#discussion_r562070442)